ibl_to_nwb/updated_conversion/datainterfaces/alfdlcinterface.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
+
+import numpy as np
+from one.api import ONE
+from pydantic import DirectoryPath
+from pynwb import H5DataIO
+from neuroconv.basedatainterface import BaseDataInterface
+from ndx_pose import PoseEstimationSeries, PoseEstimation
+
+
+class AlfDlcInterface(BaseDataInterface):
+    def __init__(self, session: str, cache_folder: DirectoryPath, camera_name: str):
+        self.session = session
+        self.cache_folder = cache_folder
+        self.camera_name = camera_name
+
+    def run_conversion(self, nwbfile, metadata: dict):
+        one = ONE(
+            base_url='https://openalyx.internationalbrainlab.org',
+            password='international',
+            silent=True,
+            cache_folder=self.cache_folder,
+        )
```

Comment:
This is very minor but I'd probably put this in the init

Reply:
I'll play around with that, it might need to be this way for certain reasons (I.e., specific sub-cache folders and persistence over multiple jobs) but I can check

It'll be this way in any of the other PRs here so I'd say ignore for now until I can determine

Reply:
Yeah it works, replacing it on other PRs now